packages/sdk/src/sdk/api/notifications/NotificationsApi.ts

```diff
@@ -0,0 +1,59 @@
+import type { Configuration } from '../../api/generated/default'
+import type { AuthService, EntityManagerService } from '../../services'
+import { Action, EntityType } from '../../services/EntityManager/types'
+import { parseParams } from '../../utils/parseParams'
+
+import {
+  MarkAllNotificationsAsViewedRequest,
+  UpdatePlaylistLastViewedAtRequest,
+  MarkAllNotificationsAsViewedSchema,
+  UpdatePlaylistLastViewedAtSchema
+} from './types'
+
+export class NotificationsApi {
+  // eslint-disable-next-line no-useless-constructor
```

Comment:
Surprised this is necessary, does it not understand the member initializations?

Reply:
yeah it doesn't :/ just copy pasta'd from the other ones... weird